src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	public String getFeedbackToUser() {
+		return feedbackToUser;
+	}
+
```

Comment:
- Did you compile after this change? TextUi.java needed a change as well.

- Keep in mind to run the regression testing once you do a change.